packages/integration_test/lib/_callback_web.dart

```diff
@@ -75,14 +75,13 @@ class WebCallbackManager implements CallbackManager {
   @override
   Future<Map<String, dynamic>> callback(
       Map<String, String> params, IntegrationTestResults testRunner) async {
-    final String command = params['command'];
+    final String command = params['command'] as String;
     Map<String, String> response;
     switch (command) {
       case 'request_data':
         return params['message'] == null
             ? _requestData(testRunner)
-            : _requestDataWithMessage(params['message'], testRunner);
-        break;
```

Comment:
Why remove this `break`?

Reply:
The line before it is a return.  It's dead code